molecule/default/prepare.yml

```diff
@@ -14,10 +14,17 @@
     - name: install aptitude for ansible
       apt: name=aptitude state=present cache_valid_time=3600
 
+    - name: Create operators group
+      group: name=operators state=present
+
     - name: create inactive user as active (to be removed)
       user:
-        name: inactiveuser
+        name: "{{ item }}"
+        groups:
+          - operators
         state: present
+      with_items: ["inactiveuser", "removeduser"]
```

Comment:
Probably also want to add a test for "existinguser" who was already in the operators group and we want to make sure they are still in the group after the play.